workers/run.go

```diff
@@ -186,7 +186,10 @@ func (r *Runner) Run(ctx context.Context, baseDir string) error {
 }
 
 func passthrough(fs *pflag.FlagSet, prefix string) (flags []string) {
-	fs.Visit(func(f *pflag.Flag) {
+	fs.VisitAll(func(f *pflag.Flag) {
+		if f.DefValue == f.Value.String() {
```

Comment:
ok, I see the problem, but I think using `f.Changed` would be better. does that work?

Reply:
Yup! works, too: `Starting worker with command: [./program --task-queue omes-test --namespace=test]`